tylerpar99/csc493-partont
# Finished issue 3 regarding route hacking.

This issue fixes #3 

Summary:
The PR stops a user from simply inputting a route and getting past the login. Now it will check that a user is logged in before sending them to any page.

To Test:
While on the `/` route try to type either `/landingPage` or `/hc/myAccount` and you will see it will return "Access Denied"
